diff_drive_controller/src/speed_limiter.cpp

```diff
@@ -58,7 +58,9 @@ namespace diff_drive_controller
     double min_acceleration,
     double max_acceleration,
     double min_jerk,
-    double max_jerk
+    double max_jerk,
+    double min_deceleration,
```

Comment:
OK, then declared them after `max_jerk` to address my other/similar comment below